dependabot-auto-merge/action.yml

```diff
@@ -9,22 +8,5 @@ inputs:
     required: true
 
 runs:
-  using: "composite"
-  steps:
-    - name: Dependabot metadata
-      id: dependabot-metadata
-      uses: dependabot/fetch-metadata@v1
-    - name: Enable auto-merge for Dependabot PRs
-      shell: bash
-      if: ${{ contains(fromJson('["version-update:semver-minor", "version-update:semver-patch"]'), steps.dependabot-metadata.outputs.update-type) && github.event.pull_request.user.login == 'dependabot[bot]' }}
-      run: gh pr merge --auto --merge "$PR_URL"
-      env:
-        PR_URL: ${{github.event.pull_request.html_url}}
-        GITHUB_TOKEN: ${{inputs.GITHUB_TOKEN}}
-    - name: Approve a PR
-      shell: bash
-      if: ${{ contains(fromJson('["version-update:semver-minor", "version-update:semver-patch"]'), steps.dependabot-metadata.outputs.update-type) && github.event.pull_request.user.login == 'dependabot[bot]' }}
-      run: gh pr review --approve "$PR_URL"
-      env:
-        PR_URL: ${{github.event.pull_request.html_url}}
-        GITHUB_TOKEN: ${{inputs.GITHUB_TOKEN}}
+  using: 'node16'
```

Comment:
should this be 18 to match .nvmrc?

Reply:
Aye, I missed this.

Reply:
I took a look at my git history and remembered why I did this - GH Action seems to support only node12 and node16, but I've never actually tested it.

I'll give it a try, as node18 is seemingly undocumented. Link to docs: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-javascript-actions

Reply:
```
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node18' is not supported, use 'docker', 'node12' or 'node16' instead.')
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
```

I should add a comment on why we use node16.